docs/tutorials/data-modeling-lifecycle/overview.md

```diff
@@ -1,13 +1,35 @@
 # Overview
 
-!!! note annotate "Warning"
-    This tutorial is a work in progress and it is not yet complete.
+!!! warning annotate "Warning"
 
+    These set of tutorials are wok in progress, and there might be minor changes to the completed ones while others
+    are being developed. We appreciate your understanding and patience.
 
-The data modeling lifecycle is a process that domain experts, information architects and solution architects use to design, build, and maintain data models. The lifecycle consists of several stages, each of which has its own set of tasks and deliverables. For each stage we have build a comprehensive tutorial on how to use **NEAT** to perform the tasks and deliver the artifacts. The stages and the tutorials are as follows:
+In this set of tutorials, we will walk you through the data modeling lifecycle and how **NEAT** can assist in the
+different tasks involved in the lifecycle. But first, let's understand what data modeling is and why it is important.
 
-- **[Knowledge Acquisition](./part-1-knowledge-acquisition.md)**: In this tutorial you will learn how to gather information about the business requirements and the data sources that will be used to build the data model from the domain experts, crafting an enterprise data model that represents the business requirements and the data sources. The enterprise data model is fine-tuned by the solution architects to ensure that it technically aligns with the organization's data infrastructure, such as Cognite Data Fusion.
+Ultimately, the goal of data modeling is to use data to aid decision-making. To achieve this, a company
+needs to coordinate between multiple different people and departments. A set of data models can be used as
+the common language to share information and provide context for data such as timeseries and documents. The
+data models can then be used as the basis for multiple different applications/solutions/reports that are used to make
+decisions. Simply put: **Data modeling is cooperation**.
 
-- **Deployment and Monitoring**: In this tutorial you will learn how to deploy the data model to a production environment and monitor it to ensure that it is performing as expected.
+In this set of tutorials, we will focus on a fictions company, `Acme Corporation`, that works in the Power & Utilities
```

Comment:
I expect you will have a better name for the company than the co-pilot suggested.

Reply:
not bad, AMCE, :) like from the cartoons